doc/classes/Image.xml

```diff
@@ -357,6 +357,7 @@
 			<param index="0" name="buffer" type="PackedByteArray" />
 			<description>
 				Loads an image from the binary contents of a JPEG file.
+				[b]Note:[/b] This method is only available in engine builds with the JPG module enabled. By default, the JPG module is enabled, but it can be disabled at build-time using the [code]module_jpg_enabled=no[/code] SCons option.
```

Comment:
```suggestion
				[b]Note:[/b] This method is only available in engine builds with the JPG module enabled. By default, the JPEG module is enabled, but it can be disabled at build-time using the [code]module_jpg_enabled=no[/code] SCons option.
```